parts/windowsProvision.ps1

```diff
@@ -1,11 +1,253 @@
 ﻿# Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-mkdir c:/bin
-mkdir c:/tmp
-cd c:/tmp
+$ErrorActionPreference = "Stop"
 
-function InstallOpenSSH()
+$AZUREDATA_DIRECTORY = Join-Path ${env:SystemDrive} "AzureData"
+$AZUREDATA_BIN_DIRECTORY = Join-Path $AZUREDATA_DIRECTORY "bin"
+$PACKAGES_DIRECTORY = Join-Path $env:TEMP "packages"
+$PACKAGES = @{
+    "openssh" = @{
+        "is_vanilla" = $true

```

Comment:
Instead of having `is_vanilla` attribute for every package, we should have a list of vanilla packages, and check against that.